docs/pages/web/iconbuttonfloating.js

```diff
@@ -0,0 +1,249 @@
+- \`accessibilityExpanded\`: informs the screen reader whether an anchored Popover-based component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+        >
+          <MainSection.Card cardSize="lg" />
+          <MainSection.Subsection
+            title="Keyboard interaction"
+            description={`
+IconButtonFloating should be contained within the \`role="contentinfo"\` container on a page. This gives screen reader users the ability to skip any main content and go directly to the action buttons. If there are multiple IconButtonFloatings, they should all be contained within the \`role="contentinfo"\` container.`}
+          />
+          <MainSection.Card cardSize="lg" />
+        </MainSection.Subsection>
+      </AccessibilitySection>
+      <MainSection name="Localization" description="Be sure to localize `accessibilityLabel`." />
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Size"
+          columns={2}
+          description="IconButtonFloating size is only available in a single size, 56px. Keeping the size consistent will promote a cohesive Pinner experience and avoid usability issues. "
+        />
+        <MainSection.Subsection
+          title="Color"
+          description={`
+          IconButtonFloating is provided in two colors schemes. It can be presented in combinations of icon and background colors.
+
+          - Dark Gray Icon ("darkGray") + White ("white") background + $elevation-floating
+          Color combination in light-mode UI.
+
+          -  White ("white") Icon + $color-background-elevation-floating
+          Color combination in dark-mode UI.
+          `}
+        />
+        <MainSection.Subsection
+          title="Elevation"
+          description={`
+          **Floating**
+          IconButtonFloating is lifted off of the background with shadows built-in ($elevation-floating). The floating elevation serves as an affordance for floating actions.
+
+          In dark mode, the elevation is achieved with colors instead of shadows ($color-background-elevation-floating). See [elevation guidelines](/foundations/elevation) for reference. `}
+        />
+        <MainSection.Subsection
+          title="Placement"
+          description={`
+          A consistent position improves discoverability as it appears across a responsive range of surfaces. In most cases, only one IconButtonFloating should be present on a screen. The exception is using a centered IconButtonFloating as a primary action, like board creation.
+
+          **Default**
+          IconButtonFloating default placement is set to stick to the bottom-end of the screen. This applies to supportive actions, such as opening related content and resources.
+
+          **Bottom-center**
+          Use bottom-center placement when leading Pinners to an action or task, such as creating a new board.
```

Comment:
This is a little awkward — I think we either need to call out `bottom-center` as a prop value (using backticks) or use an article ("Use _the_ bottom-center placement…")

Reply:
Oh wait — now I'm confused. This section reads as if we controlled placement and these are two props the user can select from. But this is just design guidance? The current format is a bit confusing to me.

Reply:
Updated this, let me know if it sounds better